### PR TITLE
Add file state

### PR DESCRIPTION
**Pull request by MarkDaoust**

### CODE REVIEW

**Review by TYMichaelChen — APPROVED** [review]
LGTM

File: google/generativeai/types/file_types.py [hunk 1]
```diff
@@ -66,6 +66,10 @@ def sha256_hash(self) -> bytes:
     def uri(self) -> str:
         return self._proto.uri
 
+    @property
+    def uri(self) -> str:
```
Comment (TYMichaelChen) [inline google/generativeai/types/file_types.py:70]: def state(self)

Comment (MarkDaoust) [reply google/generativeai/types/file_types.py:70]:
```suggestion
    def state(self) -> glm.File.State:
```

Comment (MarkDaoust) [reply google/generativeai/types/file_types.py]:
```suggestion
    def state(self) -> str:
```

**Comment by MarkDaoust** [comment]
Yeah. Fixed.

**Review by markmcd — APPROVED** [review]

**Pull request merged**